app/Filament/Resources/UserResource/Pages/Traits/HandleRole.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\UserResource\Pages\Traits;
+
+use App\Enums\Role;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Model;
+
+trait HandleRole
+{
+    private function handleRole(Model $model, string $role): void
+    {
+        /** @var User */
+        $user = auth()->user();
+
+        $validRoles = [Role::Editor->value];
```

Comment:
we need a way to add roles as current demo/prod have been seeded prior to these changes and only include superadmin/admin